Sources/Pathspec/GitIgnoreSpec.swift

```diff
@@ -8,14 +8,25 @@
 import Foundation
 
 struct GitIgnoreSpec: Spec {
+    enum Error: Swift.Error {
+        case emptyPattern
+        case containsPoundPrefix
+        case containsTrippleStar
+        case emptyRoot
```

Comment:
This one doesn't seem to be used at all

Reply:
Good catch! It was meant for this guard:

```swift
guard pattern != "/" else { throw Error.emptyRoot }
```